chain/chunks/src/lib.rs

```diff
@@ -132,43 +139,64 @@ impl RequestPool {
     }
 }
 
+#[derive(Debug, Eq, PartialEq)]
+enum Seal<'a> {
+    Past,
+    Active(&'a mut ActiveSealDemur),
```

Comment:
Why do we need to use reference here?

Reply:
We need a reference because we rely on being able to mutate the object (e.g. [here](https://github.com/nearprotocol/nearcore/blob/master/chain/chunks/src/lib.rs#L940)), but the object itself lives in the `active_demurs` hash map.